File: problemas/1100/1154.dart
import 'dart:io';

void main() {
  int n;
  var idades = new List<int>();

  while((n = int.parse(stdin.readLineSync())) >= 0) {
    idades.add(n);
  }

  stdout.writeln((idades.reduce((a, b) => a + b) / idades.length).toStringAsFixed(2));
}

File: problemas/1100/1154.js
const entrada = require('fs')
  .readFileSync('/dev/stdin', 'utf8')
  .split('\n')
  .map(x => Number(x));
let n;
const idades = [];

while ((n = entrada.shift()) >= 0) {
  idades.push(n);
}

console.log((idades.reduce((a, b) => a + b) / idades.length).toFixed(2));